InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -13947,7 +13946,7 @@ public LSL_List llJson2List(string text)
             try
             {
                 // Special case an empty string as meaning empty list
-                if (text == "")
+                if (String.IsNullOrEmpty(text))
```

Comment:
No semantic change: if str was null it would have caused a null exception already in `StripOneBOM` called a couple of lines up by `StripBOM`.